site/fs.js

```diff
@@ -0,0 +1,49 @@
+const fs = require('fs').promises
+const path = require('path')
+const rimraf = require('rimraf')
+const util = require('util')
+
+const rimrafAsync = util.promisify(rimraf)
+
+const copyDirRecursiveAsync = async (src, dest) => {
```

Comment:
We probably should be using a library instead of implementing this.
[`cpy`](https://github.com/sindresorhus/cpy) could work, but alternatives might work too.

Reply:
I didn't use one as this is a simple to implement and only really used in a single place, `./site/sync.js`.

But yeah, an external dep would be better, as there are not even tests for that implementation I wrote. Let me know if you want me to change to `cpy`.

Reply:
I agree completely, I meant simple for this use case in specific, which is using for syncing files during development, not something that is going to be shipped. I'm going to add `cpy` and use it instead inside `site/`.